query-node/mappings/src/workingGroups.ts

```diff
@@ -688,6 +691,17 @@ export async function workingGroups_StatusTextChanged({ store, event }: EventCon
   await store.save<StatusTextChangedEvent>(statusTextChangedEvent)
 }
 
+export async function contentWorkingGroups_LeadRemarked({ store, event }: EventContext & StoreContext): Promise<void> {
+  const [metadataByte] = new WorkingGroups.LeadRemarkedEvent(event).params
+
+  const leadRemarkMetadata = deserializeMetadata(LeadRemarked, metadataByte)
```

Comment:
Although `deserializeMetadata` is a recommended way of processing the metadata, there is now an inconsistency, since `MemberRemarked` still uses `MemberRemarked.decode`, so `processUpdateAppMessage` will behave differently depending on whether it's part of `MemberRemark` or `LeadRemark`.